pkg/users/git_test.go

```diff
@@ -53,6 +53,27 @@ func TestFindUserByLabel(t *testing.T) {
 	assert.Equal(t, labeleduserID, user.Spec.Login)
 }
 
+func TestFindUserByUpperCaseGitUserName(t *testing.T) {
+	t.Parallel()
+	resolver, _, err := prepare(t)
+	assert.NoError(t, err)
+	assert.NoError(t, err)
+	gituserID := "foo"
+
+	gitUser := gits.GitUser{
+		Login: gituserID,
+	}
+
+	user, err := resolver.Resolve(&gitUser)
+	assert.NoError(t, err)
+	assert.NotNil(t, user)
+
+	gitUser.Login = "Foo"
```

Comment:
Instead of just updating `Login` I'd create a new instance. Mainly for symmetry. At least for me, it gets much easier to grasp what's going on.